runtime/compiler/runtime/MetaData.cpp

```diff
@@ -176,7 +176,11 @@ createExceptionTable(
          *(uint32_t *)cursor = e->_instructionEndPC, cursor += 4;
          *(uint32_t *)cursor = e->_instructionHandlerPC, cursor += 4;
          *(uint32_t *)cursor = e->_catchType, cursor += 4;
-         if (comp->fej9()->isAOT_DEPRECATED_DO_NOT_USE())
+         if (comp->fej9()->isAOT_DEPRECATED_DO_NOT_USE()
+#if defined(JITSERVER_SUPPORT)
+            || comp->getPersistentInfo()->getRemoteCompilationMode() == JITServer::SERVER
```

Comment:
Since we have the comp object, a better test could be comp->isOutProcessCompilation()